todo-to-pr-comment-issue/src/index.test.ts

```
@@ -0,0 +1,612 @@
+import type * as github from "@actions/github";
+
+import { afterEach, beforeEach, describe, expect, it, vi } from "vitest";
```

ESLint

[error] Unable to resolve path to module 'vitest'.

(import-x/no-unresolved)